calendar/README.md

```diff
@@ -0,0 +1,73 @@
+# Calendar
+
+## 1. REST, routing
+
+Create a service that will store client events (title, description, point of time (date and time in certain timezone) of start, duration, notes [optional])
+
+### REST CRUD for events;
+    
+* GET events (with filter by date/data-range, time range, title), 
+* GET event by ID, 
+* POST create an event, 
+* PUT update an event (title, description, data & time, timezone, duration, notes), 
+* DELETE delete event.
+
+### Hints
+
+* Start from utilizing standard packages: `net/http`;
```

Comment:
https://quii.gitbook.io/learn-go-with-tests/build-an-application/app-intro
tutorial for simple http server with tests

Reply:
Thank you, added